var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -154,10 +154,10 @@ def cache_test_sources(self):
     def run_cxx_example_test(self):
         """Run stand alone test: cxx_example"""
 
-        test_dir = join_path(self.test_suite.current_test_cache_dir, 'examples', 'apps')
+        test_dir = self.test_suite.current_test_cache_dir.examples.apps
 
         if not os.path.isfile(join_path(test_dir, 'cxx-example.cpp')):
-            tty.msg('Skipping caliper test: file does not exist')
+            tty.warn('Skipping caliper test: file does not exist')
```

Comment:
LGTM.  Since you're changing this line, I suggest being more explicit:

```suggestion
            tty.warn('Skipping caliper test: cxx-example.cpp does not exist')
```

Or, better still, move `exe` above line 159 so you can use it in the warning.

Reply:
I confirm the tests work with and without the example program.  Suggest making the message a bit more specific when skipping the test in case more examples are added later.